src/main/resources/public/js/vue.min.js

```diff
@@ -0,0 +1,8 @@
+/*!
```

Comment:
I will remove vuejs, because it shouldn't be in this merge (Spring added), it was a test.
We can add webjar: http://www.webjars.org/documentation#springboot